classes/controller/FrontController.php

```diff
@@ -465,7 +472,12 @@ public function init()
         $this->context->cart = $cart;
         $this->context->currency = $currency;
 
-        Hook::exec('actionFrontControllerAfterInit');
```

Comment:
@PierreRambaud The actionFrontControllerAfterInit hook suppression is a BC break and should be documented I guess

Reply:
Yes, the hook has been renamed.